src/components-v2/InfluenceVault/InfluenceVaultUtil.tsx

```diff
@@ -0,0 +1,129 @@
+import mainnetDeploy from '../../config/deployments/mainnet.json';
+
+export const parseText = (text: string[], createLink: (a: string, b: string) => JSX.Element) => {
+	return (
+		<>
+			{text.map((t) => {
+				if (t.startsWith('[')) {
+					const name = t.substring(1, t.indexOf(']'));
+					const link = t.substring(t.indexOf('(') + 1, t.length - 1);
+					return createLink(name, link);
+				} else {
+					return t;
+				}
+			})}
+		</>
+	);
+};
+
+export const isInfluenceVault = (address: string): boolean => {
+	return address === mainnetDeploy.sett_system.vaults['native.icvx'];
+};
+
+export const getInfluenceVaultConfig = (address: string): any => {
+	return info;
+};
+
+export const getAllInfluenceVaults = (): string[] => {
+	return [mainnetDeploy.sett_system.vaults['native.icvx']];
+};
+
+export const info = {
```

Comment:
was thinking of putting this in json file in config. wanted to support the token->token config. But hardcoded addresses could work fine for the sources in chart

Reply:
can move later